java/test/src/main/java/test/Slice/generation/list-generated.out

```diff
@@ -1,15 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <generated>
   <source name="File1.ice">
-    <file name="classes/test/Slice/generation/Test/_Marker.java"/>
     <file name="classes/test/Slice/generation/Test/Interface1Prx.java"/>
     <file name="classes/test/Slice/generation/Test/_Interface1PrxI.java"/>
-    <file name="classes/test/Slice/generation/modpkg/Test2/_Marker.java"/>
     <file name="classes/test/Slice/generation/modpkg/Test2/Class1.java"/>
     <file name="classes/test/Slice/generation/Test/Interface1.java"/>
   </source>
   <source name="File2.ice">
-    <file name="classes/test/Slice/generation/Test/_Marker.java"/>
```

Comment:
The `_Marker` files are only generated when you use `java:package`.
After switching to `java:identifier` they are no longer generated in this test.